src/modules/apps/uninstall.js

```diff
@@ -14,8 +18,23 @@ const promptAppUninstall = (app) => {
   .then(({confirm}) => confirm)
 }
 
-function uninstallApps (apps = [], preConfirm) {
-  const app = apps.shift() || `${manifest.vendor}.${manifest.name}`
+function courierCallback (apps) {
+  let counter = 0
+  return () => {
+    clearLine(process.stdout, 0)
+    cursorTo(process.stdout, 0)
+    const app = apps[counter]
+    counter += 1
+    log.info(`Uninstaled app ${app} successfully`)
+    if (counter === apps.length) {
```

Comment:
I understand why this is here right now, but I'd love it to be abstracted away soon. Very much outside of the interest of the `uninstall` module. Unfortunately, no easy way for `courier` to handle it rn.